### ISSUE TITLE
gobuster update breaks automatic findings text file commands

### ISSUE THREAD

**Issue by jan0sik** [codingo/Reconnoitre#104]
Gobuster v3.0.1 now has mode options called "dns", "dir", "vhost". This will need to be reflected in the exported findings file that is generated for the automatic gobuster commands.

https://github.com/codingo/Reconnoitre/blob/758247afc3a325cbf8224927fa095fbb0065fa6b/Reconnoitre/lib/config.json#L34-L35

These need "dir" added to each.

```
gobuster dir -w /usr/share/seclists/Discovery/Web-Content/common.txt -u https://$ip:$port/ -s '200,204,301,302,307,403,500' -e | tee '$outputdir/$ip_$port_gobuster_common.txt'
gobuster dir -w /usr/share/seclists/Discovery/Web-Content/CGIs.txt -u https://$ip:$port/ -s '200,204,301,307,403,500' -e | tee '$outputdir/$ip_$port_gobuster_cgis.txt'
```

Will throw a pull request.

**Comment by codingo** [comment]
@jan0sik I believe this was resolved in #103, but very open to another pull request if not

**Comment by codingo** [comment]
Actually, I saw you submitted that. Thank-you, and closing this issue.

**Comment by codingo** [comment]
Closed with #103

**Comment by jan0sik** [comment]
Missed the other gobuster commands in the http output. I only fixed it in the https output, when merged. When you have the time, can you fix it quick?

https://github.com/codingo/Reconnoitre/blob/70c8a1909700e4ee4296541f2293908aaeeaedf4/Reconnoitre/lib/config.json#L34-L35

**Comment by codingo** [comment]
Will do!

**Comment by keithwoelke** [comment]
I don't think this was ever fixed? Also, even when adding the dir mode now, it fails because the blacklist needs to be overridden to nothing.

Error: error on parsing arguments: status-codes ("200,204,301,302,307,403,500") and status-codes-blacklist ("404") are both set - please set only one. status-codes-blacklist is set by default so you might want to disable it by supplying an empty string.
